File: Plugins/Kingbotk/Branches/Version 1/WF2/WikiFunctions/WikiFunctions2.cs
namespace WikiFunctions
{
    using Microsoft.VisualBasic.CompilerServices;
    using System;
    using System.Reflection;

    [StandardModule]
    public sealed class WikiFunctions2
    {
        public static System.Version Version
        {
            get
            {
                return Assembly.GetExecutingAssembly().GetName().Version;
            }
        }
    }
}

File: Plugins/Kingbotk/Branches/WikiFunctions2/WikiFunctions2.vb
Public Module WikiFunctions2
    ''' <summary>
    ''' WikiFunctions2.dll version
    ''' </summary>
    Public ReadOnly Property Version() As System.Version
        Get
            Return System.Reflection.Assembly.GetExecutingAssembly.GetName.Version
        End Get
    End Property
End Module
